cmd/amp/account.go

```diff
@@ -40,198 +55,759 @@ var (
 	switchRoleCmd = &cobra.Command{
 		Use:   "switch [ORGANIZATION]",
 		Short: "Switch primary organization",
+		Long:  `Switch command changes the primary organization of a resource to the specified organization.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return switchRole(AMP, cmd, args)
+		},
 	}
 
 	createOrganizationCmd = &cobra.Command{
-		Use:   "create organization [NAME] [EMAIL]",
+		Use:   "createorg [NAME] [EMAIL]",
```

Comment:
I don't like the way these are coming out ... `createorg`, `listteam`, etc. We need to think through a better command-noun-verb-attribute model, but I would think something more like: `amp account create --organization` or `amp account create --type=organization` (default `type=user`)... Let's discuss more.